logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/mention/MessageMentionMapper.kt

```diff
@@ -30,18 +30,21 @@ class MessageMentionMapperImpl(private val idMapper: IdMapper) : MessageMentionM
         )
     }
 
-    override fun fromProtoToModel(mention: Mention): MessageMention? = mention.qualifiedUserId?.let { qualifiedUserId: QualifiedUserId ->
+    override fun fromProtoToModel(mention: Mention): MessageMention {
+
         // for now we only support direct mentions which require userId https://github.com/wireapp/kalium/pull/857#discussion_r960302664
-        MessageMention(
+        return MessageMention(
             start = mention.start,
             length = mention.length,
-            userId = idMapper.fromProtoUserId(qualifiedUserId)
+            userId = mention.qualifiedUserId?.let { idMapper.fromProtoUserId(it) } ?: UserId(mention.mentionType?.value as String, ""),
```

Comment:
We shouldn't assign an empty domain, we should assume it's the same domain as the self user belongs to.